packages/rspeedy/core/test/cli/dev.test.ts

```diff
@@ -53,6 +54,27 @@ describe('CLI - dev', () => {
     vi.restoreAllMocks()
   })
 
+  test.only('config with .env file', async () => {
+    const core = await import('@rsbuild/core')
+    const { gracefulExit } = await import('exit-hook')
+
+    const program = new Command('test')
+    await dev.call(
+      program,
+      join(fixturesRoot, 'project-with-env'),
+      {},
+    )
+
+    // make sure the output contains variable BAR
+    expect(core.logger.debug).toBeCalledWith(
```

Comment:
I don't believe asserting on `logger.debug` is the most effective way to test, as it focuses on the implementation rather than the behavior.

I would prefer using `rsbuild.initConfigs()` and check if the environment variables are injected.

Reply:
I updated the test case with `initConfigs`